api/php/api.php

```diff
@@ -0,0 +1,30 @@
+<?php
+header('Expires:Mon, 26 Jul 1997 05:00:00 GMT');
+header('Last-Modified:'.gmdate('D,dMYH:i:s').'GMT');
+header('Cache-Control:no-cache, must-revalidate');
+header('Pragma: no-cache') ;
+header('Access-Control-Allow-Origin:*');
+$ip =  $_SERVER['REMOTE_ADDR'];
+
+try{
+  $db= new PDO("mysql:host=localhost;dbname=iloveu;charset=utf8mb4","root", "");
+}
+catch(PDOException $ex){
+}
+
+  $req = $db->prepare("INSERT INTO 'love'( 'IP') VALUES (':IP') ");
+  $req->execute(array(
+    "IP" => $ip
+  ));
+
+  $req = $db->query("SELECT * FROM love ");
+  $count = 0;
```

Comment:
!! on utilise du mysql pour faire un count, imagine si tu as 3000000 entrées... tu itéres dessus afin d'icrementer un compteur et cela va prendre énormément de temps alors que mysql par défaut le gère parfaitement. Par exemple (un truc du genre... je ne fais plus de pdo depuis longtemps xD) : `$count =  $DB->query('SELECT COUNT(*) FROM love')->fetchColumn();
`